docs/topics/Automated-Checks.md

```diff
@@ -0,0 +1,257 @@
+    M[Is the count of scores 0?]
+    N[Apply NoScores flag to RejectionReason]
+    O[Is the count of PreVerified and/or Verified scores 0?]
+    P[Apply NoValidScores flag to RejectionReason]
+    Q[Is the count of PreVerified and/or Verified scores 
+        twice that of the tournament's LobbySize?]
+    R[Apply LobbySizeMismatch flag to RejectionReason]
+    S[Is the ScoringType ScoreV2?]
+    T[Apply InvalidScoringType flag to RejectionReason]
+    U[Is the TeamType TeamVs?]
+    V[Apply InvalidTeamType flag to RejectionReason]
+   PreTerm[Is the RejectionReason null?]
+   TermPositive[Change VerificationStatus to PreVerified]
+   TermNegative[Change VerificationStatus to PreRejected]
+    
+    A -- Yes --> G
+    A -- No --> B
+    B -- Yes --> C
+    B -- No --> G
+    C -- Yes --> D --> E
+    C -- No --> E
+    E -- Yes --> G
+    E -- No --> F --> G
+    G -- Yes --> H --> I
+    G -- No --> I
+    I -- Yes --> J --> K
+    I -- No --> K
+    K -- Yes --> M
+    K -- No --> L --> M
+    M -- Yes --> N --> S
+    M -- No --> O
+    O -- Yes --> P --> S
+    O -- No --> Q
+    Q -- Yes --> S
+    Q -- No --> R --> S
+    S -- No --> T --> U
+    S -- Yes --> U
+    U -- No --> V --> PreTerm
+    U -- Yes --> PreTerm
+    PreTerm -- Yes --> TermPositive
+    PreTerm -- No --> TermNegative
+```
+
+### GameScore
+
+```Mermaid
+flowchart TD;
+    A[Is the score value > 1,000?]
+    B[Apply ScoreBelowMinimum flag to RejectionReason]
+    C[Does the score contain invalid mods?]
+    D[Apply InvalidMods flag to RejectionReason]
+    E[Does the ruleset match the tournament's ruleset?]
+    F[Apply RulesetMismatch flag to RejectionReason]
+    PreTerm[Is the RejectionReason null?]
+    TermPositive[Change VerificationStatus to PreVerified]
+    TermNegative[Change VerificationStatus to PreRejected]
+   
+    A -- No --> B --> C
+    A -- Yes --> C
+    C -- Yes --> D --> E
+    C -- No --> E
+    E -- Yes --> PreTerm
+    E -- No --> F --> PreTerm
+    PreTerm -- Yes --> TermPositive
+    PreTerm -- No --> TermNegative
+```
+
+## FAQ
+
+### How can a human manually mark all entities as `Verified`?
+
+Most of the issues which require manual intervention are at the `Match` and `Game` levels. For example, if a `Match` has too many invalid games, it will be marked as `PreRejected` and require manual intervention. The same is true for `Game`s.
+
+For `GameScore` entities, there are very concrete rules which can easily determine whether it should be `Rejected`, for example if the `Score` value is below the minimum (and thus very likely comes from a referee in the lobby or other anomaly).
+
+We also have a web interface which allows reviewers to mark an entity - and all of its children - as `Verified` or `Rejected`. Generally speaking, if at a glance everything is marked as `PreVerified`, very little effort is required to manually approve these submissions. If the opposite is true, it's likely that the submission contains invalid data.
```

Comment:
I'm not the best with wording but this section reads a bit odd to me

Reply:
> Most of the issues which require manual intervention are at the `Match` and `Game` levels. For example, if a `Match` has too many invalid games, it will be marked as `PreRejected` and require manual intervention. The same is true for `Game`s.

I think I dislike the usage of "require manual intervention" as if anything _doesn't_ require manual intervention? Items have to be manually verified or rejected in some capacity regardless of their existing verification status

Reply:
> For `GameScore` entities, there are very concrete rules which can easily determine whether it should be `Rejected`, for example if the `Score` value is below the minimum (and thus very likely comes from a referee in the lobby or other anomaly).

This "For `GameScore` entities, there are very concrete rules which can easily determine whether it should be `Rejected`" implies that what follows will be some sort of guide for how to review `GameScore`s, but then you describe the criteria for the `GameScoreCheck`, which results in a pre rejection?

Reply:
Basically in this section I'm attempting to state how 'easy' it is for reviewers to not have to comb through millions of scores. I'll try to reword.

Reply:
Reworded, please let me know your thoughts. @myssto